packages/blade/src/components/Button/IconButton/IconButton.tsx

```diff
@@ -34,22 +39,20 @@ type IconButtonProps = {
  * Component for making clickable icons with transparent background.
  * For other cases please use `Button` component with `icon` prop.
  */
-const IconButton = ({
-  icon,
-  onClick,
-  size = 'medium',
-  contrast = 'low',
-  accessibilityLabel,
-}: IconButtonProps): ReactElement => {
-  return (
-    <StyledIconButton
-      onClick={onClick}
-      contrast={contrast}
-      size={size}
-      icon={icon}
-      accessibilityLabel={accessibilityLabel}
-    />
-  );
-};
+const IconButton = React.forwardRef<BladeElementRef, IconButtonProps>(
+  ({ icon, onClick, size = 'medium', contrast = 'low', accessibilityLabel }, ref): ReactElement => {
+    const innerRef = useBladeInnerRef(ref);
```

Comment:
why is this required?

Reply:
To ensure we don't expose all the DOM methods from the ref, like className, style methods etc, useBladeInnerRef only exposes focus and scrollIntoView methods.

Reply:
no no I meant why are we adding ref here now?

Reply:
> no no I meant why are we adding ref here now?

Oh to do initialFocusRef. 

When bottomsheet first opens it should focus on the close button.